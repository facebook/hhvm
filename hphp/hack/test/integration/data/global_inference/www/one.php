<?hh //partial

class One {
  public function foo() {
    return 1;
  }

  public function bar () {
    return null;
  }

  public function noparameter($x) {
    return $x;
  }
}
