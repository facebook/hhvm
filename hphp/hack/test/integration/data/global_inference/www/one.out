<?hh //partial

class One {
  public function foo() : <<__Soft>> int {
    return 1;
  }

  public function bar () : <<__Soft>> ?int {
    return null;
  }

  public function noparameter(<<__Soft>> int $x) : <<__Soft>> int {
    return $x;
  }
}
